app/src/main/java/org/projectbuendia/client/providers/Contracts.java

```diff
@@ -36,7 +36,8 @@
         ORDERS("orders"),
         PATIENTS("patients"),
         USERS("users"),
-        SYNC_TOKENS("sync_tokens");
+        SYNC_TOKENS("sync_tokens"),
+        UNSYNC_TOKENS("unsync_forms");
```

Comment:
Ok, I think that's okay as long as:

1. Order execution is synchronous-only
2. Pending submissions are retried automatically when you submit the form

I think #1 is important enough that it deserves a comment explaining the design decision.  The rationale for #2 is that it would probably be confusing if a user submits some information (A) and it gets queued, and then later the user submits some more information (B) and B gets saved but A is still not saved.

@dancunningham Any thoughts on the UX for this?  Is there a document or wiki page describing the technical design or UX design for this feature?